testctrl/proto/test_sessions.proto

```diff
@@ -0,0 +1,165 @@
+// Copyright 2020 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto;
+
+option go_package = "github.com/grpc/grpc/testctrl/genproto/testctrl/svc";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/longrunning/operations.proto";
+import "src/proto/grpc/testing/control.proto";
+
+service TestSessions {
+  // Starts a set of test scenarios.
+  //
+  // This is a long-running operation that is managed by the Operations service
+  // on the same server.  The long-running operation will assign a unique
+  // identifier to the session.
+  //
+  // The unique identifier can be used to poll for the session's status and
+  // results.  Streaming is not supported at present.
+  //
+  // It can also be used to cancel while in progress, but cancellations operate
+  // like killing a running process. If they occur while results are being
+  // reported, the persistent storage may receive some but not all of them.
+  //
+  // For the specification of a google.longrunning.Operation message, see the
+  // Long-running AIP at https://aip.dev/151.
+  rpc StartTestSession(StartTestSessionRequest) returns (google.longrunning.Operation) {
+    option (google.longrunning.operation_info) = {
+      response_type: "TestSessionResults"
+      metadata_type: "TestSessionMetadata"
+    };
+  }
+}
+
+message Container {
+  enum Kind {
```

Comment:
I felt like explicitly specifying a **`Kind`** is more future proof. As a user of the API, I felt like I could instantly gauge what is going on without referring to documentation. While the current driver uses the first worker as the server, this may not always be the case. There could eventually be additional servers for more e2e performance benchmarks. The overhead of implementing a **`Kind`** enum versus relying on the first field is trivial. Thoughts?